app/R/los_distributions.R

```diff
@@ -16,26 +16,57 @@ los_zhou_general <- distcrete::distcrete("weibull", shape = 2, scale = 13, w = 0
 ## los_critical for critical care
 los_zhou_critical <- distcrete::distcrete("weibull", shape = 2, scale = 10, w = 0, interval = 1)
 
+weibull_k_value <- function(k, cv){ cv^2 - (gamma(1 + 2/k)/gamma(1 + 1/k)^2) + 1  }
+
+weibull_mucv2shapescale <- function(mean, cv){
+  params       <- list(shape = uniroot(f = weibull_k_value, interval = c(0.1, 1000), cv = cv)$root)
+  params$scale <- mean/gamma(1 + 1/params$shape)
+  params
+}
+
```

Comment:
Nice - would be a welcome addition to *epitrix* https://github.com/reconhub/epitrix